themes/bootstrap3/templates/RecordTab/holdingsils/standard.phtml

```diff
@@ -52,7 +52,11 @@
           <?php endif; ?>
         <?php else: ?>
           <?php /* Begin Unavailable Items (Recalls) */ ?>
-          <span class="text-danger"><?=$this->transEsc($holding['status'])?><link property="availability" href="http://schema.org/OutOfStock"></span>
+          <?php if ($holding['in_transit'] ?? null): ?>
+              <span class="text-danger"><?=$this->transEsc('transit_to', ['%%location%%' => $holding['in_transit'], '%%date%%' => $holding['in_transit_date']])?></span>
```

Comment:
Is it worth supporting a situation where a transit destination is available but a date is not, in case we implement this for other systems with more limited data availability? I'm thinking something like:

`<?=$this->transEsc(empty($holding['in_transit_date']) ? 'transit_to' : 'transit_to_date', ['%%location%%' => $holding['in_transit'], '%%date%%' => $holding['in_transit_date'] ?? ''])?>`

Then we could define transit_to as a language string without the date, and transit_to_date as the same string with the date added. The code will pick the appropriate string based on the presence or absence of a date value, and the added null coalescing will ensure that a missing date won't cause any errors (it will just be ignored when the 'transit_to' value is used).